Buildings/Experimental/DHC/CentralPlants/Cooling/Plant.mo

```diff
@@ -0,0 +1,383 @@
+within Buildings.Experimental.DHC.CentralPlants.Cooling;
```

Comment:
Overall, I find the diagram version of this model a little difficult to read.
Could you try to move blocks around so that most lines are either straight or have only a couple of turns, with as little crossing as possible. Make sure that blocks are centered on a grid square, and play around with rotation and mirroring so that junction are inline with connection (rather than creating an elbow).

I know that this is a very vague and subjective comment, so I won't block the PR on this, but give it a try and see if you can make this a little more easy to read.